Source/ZoomNet/Models/CallLog.cs

```diff
@@ -0,0 +1,280 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// A call log item.
+	/// </summary>
+	public class CallLog
+	{
+		/// <summary>
+		/// Gets or sets the call Id.
+		/// </summary>
+		/// <value>The call id.</value>
+		[JsonPropertyName("id")]
+		public string Id { get; set; }
+
+		/// <summary>
+		/// Gets or sets the "accepted by" information.
+		/// </summary>
+		/// <value>Indicates who accepted the call.</value>
+		[JsonPropertyName("accepted_by")]
+		public CallLogTransferInfo AcceptedBy { get; set; }
+
+		/// <summary>
+		/// Gets or sets the call answer time.
+		/// </summary>
+		/// <value>
+		/// The call's answer time, in GMT date-time format.
+		/// The API only displays this response if the direction value is inbound.
+		/// </value>
+		[JsonPropertyName("answer_start_time")]
+		public DateTime AnswerStartTime { get; set; }
+
+		/// <summary>
+		/// Gets or sets the call end time.
+		/// </summary>
+		/// <value>The call end time, in GMT date-time format.</value>
+		[JsonPropertyName("call_end_time")]
+		public DateTime CallEndTime { get; set; }
+
+		/// <summary>
+		/// Gets or sets the unique id.
+		/// </summary>
+		/// <value>Unique identifier of the phone call.</value>
+		[JsonPropertyName("call_id")]
+		public string CallId { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee country code.
+		/// </summary>
+		/// <value>Country calling code.</value>
+		[JsonPropertyName("callee_country_code")]
+		public string CalleeCountryCode { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee country iso code.
+		/// </summary>
+		/// <value>ISO alpha2 country code.</value>
+		[JsonPropertyName("callee_country_iso_code")]
+		public string CalleeCountryIsoCode { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee did number.
+		/// </summary>
+		/// <value>Callee's DID (direct inward dial) number in e164 format.</value>
+		[JsonPropertyName("callee_did_number")]
+		public string CalleeDidNumber { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee name.
+		/// </summary>
+		/// <value>The callee name.</value>
+		[JsonPropertyName("callee_name")]
+		public string CalleeName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee number.
+		/// </summary>
+		/// <value>The callee number.</value>
+		[JsonPropertyName("callee_number")]
+		public string CalleeNumber { get; set; }
+
+		/// <summary>
+		/// Gets or sets the number type of the callee.
+		/// </summary>
+		/// <value>The callee number type.</value>
+		[JsonPropertyName("callee_number_type")]
+		public int CalleeNumberType { get; set; }
```

Comment:
Rather than exposing an `int`, we should create an enum than would look something like this:
```csharp
namespace ZoomNet.Models
{
    public enum CalleeNumberType
    {
        ExtensionNumber = 1,
        PhoneNumber = 2,
        CustomizedEmergencyNumber = 3
	}
}
```